cmd/kubeadm/app/master/manifests.go

```diff
@@ -151,6 +157,22 @@ func WriteStaticPodManifests(cfg *kubeadmapi.MasterConfiguration) error {
 	return nil
 }
 
+func newVolume(cfg *kubeadmapi.MasterConfiguration, name, path string) api.Volume {
```

Comment:
all Volume function pass cfg (etc. pkiVolume).  should we remove cfg from them ?